use-case-guides/workday-request-time-off/README.md

```diff
@@ -3,129 +3,96 @@ design_pattern_id: 2
 3. What are the required fields for managers to review time off requests (example: comments, business justification). Which are optional?
+    * This affects what questions will be asked to employees by your copilot when they submit a time off request
 4. Can your employees request time off requests only for themselves, or also for someone else?
    * Note: this guide only allows employees to request time off for themselves. See API research below to understand how you can customize API research to allow time off requests for other employees
+5. Do you want employees to be notified when their time off request is approved through the bot?
+    * Note: this guide only describes how employees can request time off. Please build a new plugin to notify employees when their time off request is approved with the [Events API](https://developer.moveworks.com/creator-studio/quickstart/events/)
 
 # For Workday Administrators
 
-This document provides a step-by-step guide designed for Workday Administrators and HR Systems Engineers, detailing the procedures for creating credentials, assigning roles, and configuring workday effectively. It serves as a practical manual to ensure accurate setup and management of technical resources.
+**Level of Access Requested**:
```

Comment:
Why is this bold instead of a heading tag?

Reply:
changed to heading